src/components/PersonLink.tsx

```diff
@@ -0,0 +1,59 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../types';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
+  const params = useParams();
+  const { sex, name, born, died, fatherName, motherName, slug } = person;
+
+  const mother = motherName ? people.find(p => p.name === motherName) : null;
+
+  const father = fatherName ? people.find(p => p.name === fatherName) : null;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': params.slug === slug })}
+    >
+      <td>
+        <Link
+          className={cn({ 'has-text-danger': sex === 'f' })}
```

Comment:
Be cautious with using color to convey meaning (e.g., 'has-text-danger' for female). This could be a potential accessibility issue for color-blind users. Consider adding additional indicators or tooltips to ensure the information is accessible to all users.